todo_app/flask_config.py

```diff
@@ -6,3 +6,6 @@ class Config:
     SECRET_KEY = os.environ.get('SECRET_KEY')
     if not SECRET_KEY:
         raise ValueError("No SECRET_KEY set for Flask application. Did you follow the setup instructions?")
+
+    TRELLO_KEY= os.environ.get('TRELLO_KEY')
```

Comment:
We should warn developers that they'll need to set these in the README, and ideally also add the keys to the `.env.template` to make it clear there that they'll need to set the values as well